python/dgl/subgraph.py

```diff
@@ -34,7 +34,8 @@ def node_subgraph(graph, nodes, *, relabel_nodes=True, store_ids=True, output_de
     graph : DGLGraph
         The graph to extract subgraphs from.
     nodes : nodes or dict[str, nodes]
-        The nodes to form the subgraph. The allowed nodes formats are:
+        The nodes to form the subgraph, which cannot have any duplicate value.
```

Comment:
```
The nodes to form the subgrpah, which cannot have any duplicated values.
The result will be undefined otherwise.
```
Same for other occurrences.

Reply:
Done